src/www/app/main.ts

```diff
@@ -59,28 +60,34 @@ function createServerRepo(
     console.debug('Detected development environment, using fake servers.');
     if (repo.getAll().length === 0) {
       repo.add(
-        shadowsocksConfigToAccessKey({
-          host: '127.0.0.1',
-          port: 123,
-          method: 'chacha20-ietf-poly1305',
-          name: 'Fake Working Server',
-        })
+        SIP002_URI.stringify(
+          makeConfig({
+            host: '127.0.0.1',
+            port: 123,
+            method: 'chacha20-ietf-poly1305',
+            tag: 'Fake Working Server',
+          })
+        )
       );
       repo.add(
-        shadowsocksConfigToAccessKey({
-          host: '192.0.2.1',
-          port: 123,
-          method: 'chacha20-ietf-poly1305',
-          name: 'Fake Broken Server',
-        })
+        SIP002_URI.stringify(
+          makeConfig({
+            host: '192.0.2.1',
+            port: 123,
+            method: 'chacha20-ietf-poly1305',
+            name: 'Fake Broken Server',
```

Comment:
```suggestion
            tag: 'Fake Broken Server',
```

Reply:
Oh right - I am surprised the tests pass...

Reply:
AH I GOT LAZY lol